src/config/index.js

```diff
@@ -103,14 +105,14 @@ function Config() {
 
         cache: {
             ttl: parseIntEnv('CACHE_TTL', 24 * 60 * 60), // 24 hours
-            revalidationInterval: parseIntEnv('CACHE_REVALIDATION_INVERVAL', 10 * 60) // 10 mins
+            revalidationInterval: parseIntEnv('CACHE_REVALIDATION_INTERVAL', 10 * 60) // 10 mins
```

Comment:
Unrelated change. Please separate these in future, either by commit or PR. :pray:

Reply:
Yeah, I just stumbled across those, was already about 3 interruptions deep and didn't have the bandwidth to open another jira.  Your point is valid, though...